WindowsServerDocs/identity/ad-fs/operations/Configure-intranet-forms-based-authentication-for-devices-that-do-not-support-WIA.md

```diff
@@ -55,11 +55,11 @@ You can add Chrome or other user agents to the AD FS configuration that supports
 
 In AD FS configuration, add a user agent string for Chrome on Windows-based platforms:
 
-    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT")
+    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT"))
 
 And similarly for Chrome on Apple macOS, add the following user agent string to the AD FS configuration:
 
-    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Macintosh; Intel Mac OS X")
+    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Macintosh; Intel Mac OS X)")
```

Comment:
... and if you said before is correct, that PowerShell doesn't like double quotation marks there, this one needs amending too:
```suggestion
    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + Mozilla/5.0 (Macintosh; Intel Mac OS X))
```

Reply:
We need the quotes to enclose the string related to the entry to add to the property. The initial problem that I've reported was about the last quote.

Reply:
Hmm, that is odd, wouldn't there be a need for a "closing" quote after the text string?

Reply:
Sorry, I have probably misunderstood the placing of the quote characters. Should it look like your original quotation usage in both places?

Reply:
Yes. I have tested it in a real environment. In fact I raised this pull request right after noticing that there was an error related to the quotes.